src/main/java/com/passbook/sparkeighteen/peristence/POJO/ProfileRequest.java

```diff
@@ -0,0 +1,25 @@
+package com.passbook.sparkeighteen.peristence.POJO;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class ProfileRequest {
+    @NotBlank(message = "mobile number is mandatory")
+    private String mobileNumber;
+
+    @NotBlank(message = "address is mandatory")
+    private String address;
+
+    private String pan;
+
+    @NotBlank(message = "aadhar number is mandatory")
+    private String aadhar;
```

Comment:
regex?

Reply:
Validation done, we followed this "xxxx xxxx xxxx"  pattern